deploy/sre-authorization/01-osd-sre-admin.ClusterRole.yaml

```diff
@@ -64,14 +72,6 @@ rules:
   - projects
   verbs:
   - '*'
-- apiGroups:
-  - config.openshift.io
-  resources:
-  - clusterversions
-  verbs:
-  - get
-  - list
-  - update
```

Comment:
Why remove `update` ?

Reply:
Not intentional at the time, but I have an answer now.  Operator version management should come from `hive` or `managed-operator`.  If we need to manipulate CSVs it will be `create` and maybe `delete`, not `update.